src/containers/AsideNavigation/AsideNavigation.tsx

```diff
@@ -184,28 +197,57 @@ function AsideNavigation(props: AsideNavigationProps) {
     return (
         <React.Fragment>
             <AsideHeader
-                logoText="YDB"
-                logoIcon={ydbLogoIcon}
-                onLogoIconClick={() => history.push('/')}
+                logo={{
+                    text: 'YDB',
+                    icon: ydbLogoIcon,
+                    onClick: () => history.push('/'),
+                }}
                 menuItems={menuItems}
-                settings={<UserSettings />}
-                initIsCompact
+                compact={props.compact}
+                onChangeCompact={(compact) => {
+                    props.setSettingValue(
+                        ASIDE_HEADER_STORE_KEY,
+                        JSON.stringify({isCompact: compact}),
+                        true,
```

Comment:
`@gravity-ui/navigation` doesn't save collapse state, delegating it to the service
we want to keep using existing saved states
so this mimics local storage entry used by the removed component